test/remote/gateways/remote_s5_test.rb

```diff
@@ -0,0 +1,136 @@
+require 'test_helper'
+
+class RemoteS5Test < Test::Unit::TestCase
+  def setup
+    @gateway = S5Gateway.new(fixtures(:s5))
+
+    @amount = 100
+    @credit_card = credit_card('4000100011112224')
+    @declined_card = credit_card('4000300011112220')
+
+    @options = {
+      order_id: '1',
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+  end
+
+  def test_dump_transcript
```

Comment:
I'm not sure how others feel about this, but I've been deleting this test method after it's served its purpose and generated a transcript I can use.  I'd whack it.